src/templates/login/login.tsx

```diff
@@ -51,7 +72,11 @@ export const LoginTemplate: React.FC<LoginTemplateProps> = ({
         <Body>
           {slotLogo || CustomLogo || <Logo />}
           <Hr />
-          <Form labelSubmit={ucFirst(_labelLogin)} {...formProps} />
+          <Form
+            labelSubmit={ucFirst(_labelLogin)}
+            secondaryActions={secondaryActions}
+            {...formProps}
```

Comment:
I'm wondering if it should be possible to overrule the default login and only allow OIDC?

Reply:
I don't know! But could be useful :thinking: